force-di/main/classes/di_PlatformCache.cls

```diff
@@ -107,91 +247,62 @@ public with sharing class di_PlatformCache
         // hashcode() is a more expensive operation than map.get()
         if (generatedKeyNames.containsKey(key)) { return generatedKeyNames.get(key); }
 
         String hash = String.valueOf( Math.abs( ( key ).hashcode() ) );
-        //System.debug('Creating Hash For => ' + developerName + ' && ' + bindingSObjectType + ' := ' + hash);
+        //log('Creating Hash For => ' + developerName + ' && ' + bindingSObjectType + ' := ' + hash);
         generatedKeyNames.put(key, hash);
         return hash;
     }
-
+    @TestVisible
     private String getKeyName( String developerName, Schema.SObjectType bindingSObjectType)
     {
         return constructKeyName( bindingSObjectType, developerName);
     }
-
+    @TestVisible
     private String getKeyName( di_Binding binding )
     {
         return constructKeyName( binding.bindingObject, binding.developerName);
     }
-
+    @TestVisible
     private String getKeyIndexName()
     {
-        return 'bindingKeyIndex';
+        return di_PlatformCache.BINDING_KEY;
     }
 
-    public void addBindingToPlatformCache( di_Binding binding )
-    {
-        if ( isStoringBindingInPlatformCache() )
-        {
-            String theKeyName = getKeyName(binding);
-            // add the binding to the platform cache directly
-            getPartition().put(theKeyName, binding, getPartitionTTL(), Cache.Visibility.ALL, false);
-            // add the binding's cache key name to the bindingKeyIndex
-            addBindingToKeyIndex(binding);
-            //System.debug('Adding binding for hash => ' + theKeyName + ' && developerName => ' + binding.developerName + ' && object => ' + binding.bindingObject);
-        }
-    }
-
-    public list<di_Binding> retrieveBindings(String developerName, Schema.SObjectType bindingSObjectType)
-    {   
-        list<di_Binding> bindings = new list<di_Binding>();
-
-        if ( isStoringBindingInPlatformCache() )
-        {
-            //System.debug('Retrieving from Cache Key => ' + developerName + ' && Cache Key Index => ' + bindingSObjectType);
-
-            Map<Schema.SObjectType, Set<String>> keyIndexBySObjectTypeMap = getCacheKeyIndexMap().get(developerName.toLowerCase().trim());
-            if ( keyIndexBySObjectTypeMap != null )
-            {
-                Set<String> cacheKeys = keyIndexBySObjectTypeMap.get(bindingSObjectType);
-                if ( cacheKeys != null )
-                {
-                    Object cachedObject = null;
-                    for ( String cacheKey : cacheKeys )
-                    {
-                        cachedObject = getPartition().get( cacheKey );
-                        if ( cachedObject != null ) 
-                        {
-                            bindings.add( (di_Binding) getPartition().get( cacheKey ) );
-                        }
-                    }
-                }
-            }
+    /**
```

Comment:
The concept of configurable, logging functionality is greatly appreciated, although my first thought is it should be it's own classes -- separation of concerns; S in SOLID.

Second thought referencing my comments in other classes, the logging functionality may be used by other classes too.

Third thought, should the configuration be extended to enable activation for non-test code -- custom setting, custom-metadata record, else?  An issue could exist within the context of normal business logic that may need more extensive logging.

Reply:
@stohn777
It seemed to me that there is no more issue for this one above. Correct?
Thanks! :)